src/System.IO/pkg/aot/System.IO.pkgproj

```diff
@@ -7,9 +7,6 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\System.IO.csproj">
-      <TargetGroup>netstandard13aot</TargetGroup>
```

Comment:
Yes, something like that ... @joshfree or @SedarG may have more detailed idea about the plan.